src/ruby/puppetserver-lib/puppet/server/master.rb

```diff
@@ -60,6 +60,10 @@ def handleRequest(request)
         response["X-Puppet-Version"])
   end
 
+  def compileCatalog(_request_data)
```

Comment:
What is the reason for the `_` for `request_data`?

Reply:
signifies that it's unused (for now)